lib/diagnostic.js

```diff
@@ -24,7 +36,13 @@ const sum = function sum() {
 // if invoking the callback for any element
 // in the array returns a truthy value
 const any = function any(array, callback) {
+  for (let i =0; i < array.length; i++) {
+    if (!callback(array[i], i, array)) {
+      return false;
+    }
+  }
 
+  return true;
```

Comment:
Close, but not quite. As written, this code will return `false` if any elements don't pass the condition. If we were to call `any` on the array `[1,3,6]`, and we had a callback that returned `true` is a number were even, this function would read the first element (`1`) and when the test came back false it would return `false` from the function. Clearly, though, this array contains an even element.